srcpkgs/jamulus/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+archs="x86_64* i686"
```

Comment:
Why restrict the archs?

Reply:
The CI fails on other archs otherwise. I could get rid of this to display the errors if you'd like.

Reply:
Fix the cross-build.

Reply:
Also, if its not possible to cross-build, use `nocross`, don't restrict the arches.

Reply:
`nocross="Compilation fails on some archs"`?

Reply:
Set `nocross` after `checksum`